src/frameworks/react/hooks.ts

```diff
@@ -82,11 +85,16 @@ export const useElectricQuery = (query: Query, params?: BindParams) => {
     }
 
     const paramsKey = JSON.stringify(params)
-    const tablenames = electric.adapter.tableNames(query)
+    const tablenames = electric.adapter.tableNames({ sql: query })
     const tablenamesKey = JSON.stringify(tablenames)
 
+    // console.log('setParamsKey', paramsKey)
```

Comment:
I think I removed these logging statements. Suggest let's strip them out.

Reply:
We need rigorous logging. We still need to think of a strategy that is clean.

- Satellite stuff I was thinking of logging into the database
- Client/Driver stuff do we want not to have any kind of logging? Do we need logging levels? Just output to console?

Removing all commented log lines